airflow/www/security.py

```diff
@@ -340,8 +340,8 @@ def clean_perms(self):
         pvms = (
             sesh.query(sqla_models.PermissionView)
             .filter(or_(
-                sqla_models.PermissionView.permission == None,  # noqa pylint: disable=singleton-comparison
-                sqla_models.PermissionView.view_menu == None,  # noqa pylint: disable=singleton-comparison
+                sqla_models.PermissionView.permission is None,  # noqa pylint: disable=singleton-comparison
```

Comment:
Are you sure? SQL Alchemy behaves differently when comparing types.  This is not used to compare types in Python, but creates a query for SQL Alchemy.

Reply:
Yes, we should still not be checking this for equality.

Reply:
Can you complete the tests to prevent regression?

Reply:
This is not right:

```
In [5]: type(TI.state is None)
Out[5]: bool

In [6]: type(TI.state == None)
Out[6]: sqlalchemy.sql.elements.BinaryExpression
```

I fyou want to avoid the wrong-looking `== None` the (`is_`)[https://docs.sqlalchemy.org/en/11/core/sqlelement.html#sqlalchemy.sql.operators.ColumnOperators.is_] function is the way to go.